### DISCUSSION TITLE
Simplest approach to execute a typer program / execute without a filename?

### DISCUSSION THREAD

**Discussion by fzyzcjy** [fastapi/typer#1319] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

Hi thanks for the lib! I wonder what is the simplest approach to execute a typer program? Or, can we execute without a filename?

Currently I know we can `python myfile.py some_command`. We can also do `typer myfile.py run some_command`.

However, looking at similar projects, such as the Justfile project, we can do `just some_command` to directly execute that command. 

Thus, I would appreciate it if we could run `typer some_command` directly. To implement this, we can follow some convention to find which python file to execute - such as a file called `typer_entrypoint.py` and so on.

### Operating System

Other

### Typer Version

latest

### Python Version

latest

**Comment by YuriiMotov** [comment, marked as answer]
You can package your app, and after installing it you will be able to use it with just `$ your_app_name [COMMAND]`

Docs: https://typer.tiangolo.com/tutorial/package/#add-a-script
